### DISCUSSION TITLE
Suggest or wishlist: Text File or TWIG output for streamers to display data on twitch etc.

### DISCUSSION THREAD

**Discussion by korruptcow** [MobiFlight/MobiFlight-Connector#414] [Ideas]
I know this not really what mobiflight is for but i figure it would be simple to implement as most of the work has been down for getting data from simconnect. At the moment, if we stream we have to use several different software to display a html overlay that updates every second on a stream. 

I was using simtoolkit pro but the issue there is its very buggy. I figure if mobiflight could output text files at minimum of variables even ever it updates. it could be used to generate an overlay for streamers. which could bring more people to use it.

I am trying to write my own in python and doing horrible. 

This is an output from STKit so you can see what you can output. It would be awesome to get any output you are using in the config and output it to text to make the templates.

![image](https://user-images.githubusercontent.com/30361335/125923233-ea649082-dd63-4518-9790-6e5b5de0ac41.png)

**Comment by MobiFlight-Admin** [comment]
i can immediately think of two ways that might be helpful: 
1) have a client for the MobiFlight WASM Module that can publish value via a REST API or websockets. This would allow other tool builders to use a very common way to access this data in their applications and it would also allow for web based sim interfaces to display gauges and stuff

2) we could create a new output type for mobiflight to send the data to a rest endpoint, or to publish it via an API as described in 1)

**Reply by korruptcow** [reply]
That would be great.
